code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm

```diff
@@ -13,6 +13,8 @@
 	var/stealth = FALSE
 	var/can_sneak_attack = FALSE
 	var/stealth_alpha_multiplier = 1
+	///Minimum damage to take at once to knock the xeno out of stealth
+	var/damage_break_threshold = 15
```

Comment:
This should be on xeno caste datum, its of no use in here.

Reply:
its on the ability datum @SplinterGP

Reply:
Yeah what use being in here? For it to matter youd have to make another ability typepath this number never changes atleast i dont see it changing